DMCompiler/DMStandard/_Standard.dm

```diff
@@ -81,6 +81,7 @@ proc/rgb(R, G, B, A, space) as text|null
 proc/rgb2num(color, space = COLORSPACE_RGB) as /list
 proc/roll(ndice = 1, sides) as num
 proc/round(A, B) as num
+proc/sign(A) as num
```

Comment:
This comes after `shutdown()` alphabetically. Move it in the other two files too.